etcdserver/api/v3rpc/watch.go

```diff
@@ -87,6 +96,16 @@ func sendLoop(stream pb.Watch_WatchServer, watchStream storage.WatchStream, clos
 				return
 			}
 
+		case c, ok := <-ctrlStream:
+			if !ok {
+				return
+			}
+
+			err := stream.Send(c)
+			if err != nil {
```

Comment:
We do not log client side error. Stream is a gRPC stream. If there is an error, then it is a client side connection issue.